flowcell_parser/classes.py

```diff
@@ -74,13 +78,16 @@ def parse(self, demultiplexingDir='Demultiplexing'):
         except OSError as e:
             self.log.info(str(e))
             self.undet=None
-
         try:
             self.time_cycles = CycleTimesParser(cycle_times_log)
         except OSError as e:
             self.log.info(str(e))
             self.time_cycles = None
-
+	try:
+            self.stats_json = json.load(json_path)
```

Comment:
in general I would prefer to keep symmetry in the code even if might sound strange. In this case I would prefer to see
`self.stat:json = StatJsonParser(json_path)`

and have a silly implementation of `StatJsonParser`